### ISSUE TITLE
no-empty rule broken

### ISSUE THREAD

**Issue by bartlomieju** [denoland/deno_lint#109]
```
(no-empty) Empty block statement
  --> ../deno/std/encoding/toml.ts:13:56
   |
13 |   constructor(public type: string, public name: string) {}
   |                                                         ^^
   |
```

**Comment by disizali** [comment]
also : 
```ts
function x(): number {}
```

```shell
(no-empty) Empty block statement
 --> ./problem.ts:1:20
  |
1 | function x():number {}
  |                     ^^
  |
(no-empty-function) Empty functions are not allowed
 --> ./problem.ts:1:0
  |
1 | function x():number {}
  | ^^^^^^^^^^^^^^^^^^^^^^
  |
Found 2 problems
```

eslint only warns about `no-empty-function`
